coderd/notifications/notifications_test.go

```diff
@@ -1933,10 +1940,11 @@ func TestNotificationDuplicates(t *testing.T) {
 		map[string]string{"initiator": "danny"}, "test", user.ID)
 	require.NoError(t, err)
 
-	// WHEN: the second is enqueued, the enqueuer will reject the request.
+	// WHEN: the second is enqueued, the enqueuer will reject it as a duplicate.
```

Comment:
Tests look good, but now that we're possibly returning a slice of UUIDs we should check the length of those slices are expected.